.cirrus.yml

```diff
@@ -1,11 +1,13 @@
 environment:
     CCACHE_BASEDIR: $CIRRUS_WORKING_DIR
+    ZEEK_PACKAGES_CURRENT: zeek-rc zeek-rc-core-dev
+    ZEEK_PACKAGES_LTS:     zeek-lts zeek-lts-core-dev
```

Comment:
I think it would be better to explicitly pin versions here, i.e., instead of e.g., `zeek-lts` install `zeek-lts=X.Y.Z`. Otherwise CI would be using whatever version was present in the repo when the (cached) image was last built which could lead to non-reproducible builds.

Similarly, we could pin versions of packages installed through pip in the `Dockerfile`s as well, but the syntax is [slightly more arcane](https://github.com/hadolint/hadolint/wiki/DL3013).